src/command_modules/azure-cli-acs/setup.py

```diff
@@ -31,7 +31,7 @@
 
 DEPENDENCIES = [
     'azure-mgmt-authorization==0.30.0rc6',
-    'azure-mgmt-compute==2.0.0',
```

Comment:
Actually it does: https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py#L34